app/components-react/pages/onboarding/Onboarding.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import styles from './Onboarding.m.less';
+import commonStyles from './Common.m.less';
+import { Services } from 'components-react/service-provider';
+import { useModule } from 'components-react/hooks/useModule';
+import cx from 'classnames';
+import { mutation } from 'components-react/store';
+import { $t } from 'services/i18n';
+import * as stepComponents from './steps';
+import Utils from 'services/utils';
+import { ONBOARDING_STEPS } from 'services/onboarding';
+
+export default function Onboarding() {
+  const { currentStep, next, processing } = useModule(OnboardingModule).select();
+  const Component = stepComponents[currentStep.component];
+
+  return (
+    <div className={styles.onboardingContainer}>
+      {<TopBar />}
+      <div className={styles.onboardingContent}>
+        {/* TODO: Add back in scrollable */}
+        {/* <Scrollable className={styles.scroll}> */}
```

Comment:
Need to test whether this scrollable actually needs to be here and make a decision.